modules/base/include/modules/base/processors/datasource.h

```diff
@@ -44,24 +44,25 @@
 namespace inviwo {
 
 namespace util {
-inline void updateReaderFromFile(const FileProperty& file, OptionProperty<FileExtension>& reader) {
+inline void updateReaderFromFile(const FileProperty& file,
+                                 OptionProperty<FileExtension>& extReader) {
     if ((file.getSelectedExtension() == FileExtension::all() &&
-         !reader.getSelectedValue().matches(file)) ||
+         !extReader.getSelectedValue().matches(file)) ||
         file.getSelectedExtension().empty()) {
-        const auto& opts = reader.getOptions();
+        const auto& opts = extReader.getOptions();
         const auto it = std::find_if(opts.begin(), opts.end(),
                                      [&](const OptionPropertyOption<FileExtension>& opt) {
                                          return opt.value_.matches(file.get());
                                      });
-        reader.setSelectedValue(it != opts.end() ? it->value_ : FileExtension{});
+        extReader.setSelectedValue(it != opts.end() ? it->value_ : FileExtension{});
     } else {
-        reader.setSelectedValue(file.getSelectedExtension());
+        extReader.setSelectedValue(file.getSelectedExtension());
     }
 }
 
 template <typename... Types>
 void updateFilenameFilters(const DataReaderFactory& rf, FileProperty& file,
-                           OptionProperty<FileExtension>& reader) {
+                           OptionProperty<FileExtension>& extReader) {
```

Comment:
I don’t understand why the name should be changed.

Reply:
Since this here refers to list entries tying an extension to a certain reader.

Reply:
But it is not an extension reader, which I think that the name implies. Would not `extensions` be more appropriate if you want to change the name?